### PR TITLE
onetbb@2022.0.0

### PR DESCRIPTION
**Pull request by c8ef**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @Vertexwahn, modules you maintain (onetbb) have been updated in this PR. Please review the changes.

File: modules/onetbb/2022.0.0/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+matrix:
+  platform:
+    - debian11
+    - macos
+    - macos_arm64
+    - ubuntu2004
+    - ubuntu2204
```
Comment (Vertexwahn) [inline modules/onetbb/2022.0.0/presubmit.yml:7]: You can add `ubuntu2404` to the list

File: modules/onetbb/metadata.json
Comment (Vertexwahn) [inline modules/onetbb/metadata.json:1]: You can add yourself to the maintainer list, if you like

**Review by Vertexwahn — APPROVED** [review]

**Review by bazel-io — APPROVED** [review]
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.

**Pull request merged**